ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -95,10 +87,22 @@ public static List<string> GetAllData()
         /// Checks if specified name exists in User table
         /// </summary>
         /// <param name="name">The name.</param>
-        /// <returns></returns>
+        /// <returns>true if name exists in local table</returns>
         public static bool Exists(string name)
         {
             return GetAllData().Contains(name);
         }
+
+        private static SqliteDataReader ExecuteCommand(SqliteCommand command)
+        {
+            return command.ExecuteReader();
+        }
+
+        private static string databaseName = "coimbra.db";
```

Comment:
This can be `const`, which will be more efficient as it will be compiled directly into the binary.